reference/5.1/Microsoft.PowerShell.Core/About/about_Quoting_Rules.md

```diff
@@ -316,8 +305,8 @@ xmlns:dev="http://schemas.microsoft.com/maml/dev/2004/10">
 ```
 
 Here-strings are also a convenient format for input to the
-ConvertFrom-StringData cmdlet, which converts here-strings to hash tables. For
-more information, see ConvertFrom-StringData.
+`ConvertFrom-StringData` cmdlet, which converts here-strings to hash tables.
+For more information, see `ConvertFrom-StringData`.
```

Comment:
Shouldn't this one be a link to that cmdlet?

Reply:
It's linked in the SEE ALSO that immediately follows